src/open/clients/Element.js

```diff
@@ -29,24 +30,12 @@ const trustedWebInstances = [
 /**
  * Information on how to deep link to a given matrix client.
  */
-export class Element {
-    get id() { return "element.io"; }
-
-    get platforms() {
-        return [
-            Platform.Android, Platform.iOS,
-            Platform.Windows, Platform.macOS, Platform.Linux,
-            Platform.DesktopWeb
-        ];
+export class Element extends Client {
+    constructor() {
+        super(data);
     }
 
-    get icon() { return "images/client-icons/element.svg"; }
     get appleAssociatedAppId() { return "7J4U792NQT.im.vector.app"; }
```

Comment:
Since Element is the only client to use it, I'm leaving the function in `Element.js` and not adding it to `Clients.js`

Reply:
might as well put it in the static data given it's just a string?